### ISSUE TITLE
Some problems come to me when compiling your code

### ISSUE THREAD

**Issue by Linspacerational** [Yangzi-Chen2023/GC_NC-Res#1]
![image](https://github.com/user-attachments/assets/78e1993b-b613-45b5-b38c-378485f1961e)
It seems that I encountered Version incompatibility issues.
Could you declare the versions of your python packages?

**Comment by Yangzi-Chen2023** [comment]
The following are the versions of Python packages:
numpy==1.26.4
scikit-learn==1.0.2
scikit-survival==0.17.0
eli5==0.11.0
pandas==2.1.0

**Comment by Linspacerational** [comment]
Thanks for your reply! And what about the detailed version of your python?

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Yangzi-Chen2023** [comment]
Python==3.9.10
